examples/scripts/spm_NelderMead.py

```diff
@@ -60,9 +60,9 @@ def noise(sigma):
     verbose=True,
     allow_infeasible_solutions=True,
     sigma0=0.05,
+    max_iterations=100,
+    max_unchanged_iterations=45,
```

Comment:
Aligning with other examples.

```suggestion
    max_unchanged_iterations=20,
```